pynq/pl.py

```diff
@@ -683,6 +687,19 @@ def __init__(self):
             raise EnvironmentError('Root permissions required.')
 
 
+def _stop_server():
+    try:
+        PL.client_request()
+        PL.server_update(0)
+    except:
+        pass
+
+def _start_server():
```

Comment:
@PeterOgden   Can we add 2 or 3 sentences as docstrings so users know what those 2 functions are doing?